cg/store/api/findbasicdata.py

```diff
@@ -89,9 +93,17 @@ def customers(self) -> List[Customer]:
         """Fetch all customers."""
         return self.Customer.query
 
-    def collaboration(self, internal_id: str) -> Collaboration:
+    def _get_collaboration_query(self) -> Query:
+        """Returns a collaboration query"""
+        return self.Collaboration.query
+
+    def get_collaboration_by_internal_id(self, internal_id: str) -> Collaboration:
```

Comment:
@seallard, @ChrOertlin @henrikstranneheim  So what's our opinion on this naming? Should it be `get_first_collaboration_by_internal_id `?

Reply:
I agree 👍

Reply:
I see, so then we resolve to that callers then contain either singular (first) or plural (all) nouns?

Reply:
👍